src/resources/js/parts-kit.js

```diff
@@ -181,6 +133,141 @@ class Toggle {
     };
 }
 
-// Init the toggle class
-const toggles = document.querySelectorAll("[data-parts-kit-toggle]");
-toggles.forEach((el) => new Toggle(el));
+// Parts kit class for expanding/collapsing sidebar
```

Comment:
Had to move the PartsKit class below the `Toggle` class so we could use the Toggle instances in our search function (opening / closing groups that have results).